staging/src/k8s.io/kubectl/pkg/cmd/auth/auth.go

```diff
@@ -19,7 +19,6 @@ package auth
 import (
```

Comment:
can revert this file since there are no actual changes

Reply:
done